### PR TITLE
[zh]some things is outdate.

### PR DESCRIPTION
**Pull request by yuandongx**

### CODE REVIEW

File: content/zh/docs/concepts/containers/images.md [hunk 1]
```diff
@@ -244,26 +244,24 @@ example, run these on your desktop/laptop:
...
-1. Run `docker login [server]` for each set of credentials you want to use.  This updates `$HOME/.docker/config.json` on your PC.
-1. View `$HOME/.docker/config.json` in an editor to ensure it contains just the credentials you want to use.
-1. Get a list of your nodes; for example:
+   1. Run `docker login [server]` for each set of credentials you want to use.  This updates `$HOME/.docker/config.json` on your PC.
+   1. View `$HOME/.docker/config.json` in an editor to ensure it contains just the credentials you want to use.
+   1. Get a list of your nodes; for example:
       - if you want the names: `nodes=$( kubectl get nodes -o jsonpath='{range.items[*].metadata}{.name} {end}' )`
       - if you want to get the IP addresses: `nodes=$( kubectl get nodes -o jsonpath='{range .items[*].status.addresses[?(@.type=="ExternalIP")]}{.address} {end}' )`
-1. Copy your local `.docker/config.json` to one of the search paths list above.
+   1. Copy your local `.docker/config.json` to one of the search paths list above.
```
Comment (tengqm) [inline content/zh/docs/concepts/containers/images.md:252]: The above indentation are stupid, Remove the indentation is the right style.

File: content/zh/docs/concepts/containers/images.md [hunk 2]
```diff
@@ -300,6 +298,26 @@ EOF
...
+
+```shell
+kubectl logs private-image-test-1
+```
+and see that the command outputs:
+```
+SUCCESS
+```
+-->
+如果都运行起来了，接下来，一段时间后你可以执行：
```
Comment (tengqm) [inline content/zh/docs/concepts/containers/images.md:312]:
```suggestion
如果一切顺利，那么一段时间后你可以执行：
```

**Comment by tengqm** [comment]
Thanks.
/approve

**Review by tanjunchen — COMMENTED** [review]
/lgtm

**Pull request merged**